File: Native/Test/Python/mr-value-generator.py
#
#  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
#

import fabric

client = fabric.createClient()

print(
  client.MR.createValueGenerator(
    client.KLC.createValueGeneratorOperator(
      "xfo.kl", "operator xfo(io Integer value) { value = 121; }", "xfo"
      )
    ).produce()
  )

client.close()

File: Native/Test/Node/mr-value-generator.js
/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

FC = require('Fabric').createClient();

console.log(
  FC.MR.createValueGenerator(
    FC.KLC.createValueGeneratorOperator(
      "xfo.kl", "operator xfo(io Integer value) { value = 121; }", "xfo"
      )
    ).produce()
  );

FC.close();
